src/alt.js

```diff
@@ -450,9 +451,14 @@ class Alt {
     }, exportObj)
   }
 
-  takeSnapshot() {
-    const state = snapshot(this)
-    this[LAST_SNAPSHOT] = state
+  takeSnapshot(...storeNames) {
+    const state = snapshot(this, ...storeNames)
+    if(this[LAST_SNAPSHOT]) {
```

Comment:
My reasoning for this was because of the following scenario.

1. I take a snapshot of my app, containing 5 stores or so.
2. Some change occur in my stores.  If I rolled back from here I would be back at my first snapshot. So far so good.
3. I take a partial snapshot of only 2 of my stores. 
4. More changes occur in my stores. If I rollback from here, the 2 stores I snapshotted will rollback, but my other stores will remain untouched.  This could be confusing behavior.  If store 5 (a store not in my second snapshot) had an error, so I try to rollback from there since I had taken a snapshot of it at some point, it would not recover back to that snapshot.  I am using assign to merge snapshots such that any stores that were in a previous snapshot, but not included in the new snapshot will not have their rollback data lost.

I understand what you are saying and would like it to always be predictable.  The problem is you could lose other store's rollback data.  This might be better solved in another ticket to make rollbacks more robust.  Curious to hear your thoughts on this scenario though.